### DISCUSSION TITLE
Is the neovim download in snap corrupted?

### DISCUSSION THREAD

**Discussion by derick-rive** [neovim/neovim#28049] [Q&A]
When I installed nvim, no problems arose, but when I started configuring it, it does not find the files that I have made.

This is my way of classifying the files:
![0](https://github.com/neovim/neovim/assets/134315002/fc8f8a64-5a44-4e23-87ba-f6d03db9fcd6)
and this is the init.lua(how the file is opened from ~/.config/nvim, the init.lua config works)
![init](https://github.com/neovim/neovim/assets/134315002/ad422561-6850-4ac9-9894-d1d6b499a20b)
When I start nvim, I get this error, which indicates that it cannot find the module
![error](https://github.com/neovim/neovim/assets/134315002/8471d65e-db15-4e7e-973c-ccc9dd421108)
I honestly don't know what goes wrong, but that's not all, when I start nvim in the ~/.config/nvim directory, the init.lua file works wonders
![neo00](https://github.com/neovim/neovim/assets/134315002/c9396e89-dc58-4495-bc38-e39d5e5ee977)
but there is another problem, the lazy package manager does not find the plugins that I have stored in the directory ~/.config/nvim/plugins, it indicates that it did not find any files, when there are
![neo](https://github.com/neovim/neovim/assets/134315002/8f42f95b-d7e9-4437-8d48-33a7694cd1c5)
![0](https://github.com/neovim/neovim/assets/134315002/0a71231d-6cb4-4104-8d71-e05b64f9797b)
...what do you recommend I do? Will it be a problem with the nvim or lazy code? Or will it be my Linux distribution (I use Kali)

I have tried installing some well-known configurations such as astronvim, lunarvim, and lazyvim and all of them work wonderfully, it does not cause problems for me and the configurations work well

**Comment by clason** [comment]
You can only `require` Lua modules under `lua`, it will look nowhere else. Please read `:h lua-guide`.
